Workarounds/BreathOfTheWild_Nvidia_Workaounds_OGL/rules.txt

```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Workarounds for Nvidia GPUs"
+path = "The Legend of Zelda: Breath of the Wild/Workarounds OpenGL/Nvidia GPU"
+description = Contains these workaounds for BOTW and Nvidia GPUs,Nvidia Smoke, Kakariko Torch Shadows, Streched Clouds, LWZXNullCheck.
```

Comment:
```
name = "Workarounds for NVIDIA GPUs"
path = "The Legend of Zelda: Breath of the Wild/Workarounds/NVIDIA GPU for OpenGL"
```
`Contains these workarounds for BOTW and NVIDIA GPUs: NVIDIA Smoke, Kakariko Torch Shadows,` etc...